src/main/java/pl/edu/zut/mad/schedule/CalendarController.kt

```diff
@@ -0,0 +1,32 @@
+package pl.edu.zut.mad.schedule
+
+import com.ognev.kotlin.agendacalendarview.CalendarController
+import com.ognev.kotlin.agendacalendarview.models.IDayItem
+import org.joda.time.LocalDate
+import java.util.*
+
+
+internal class CalendarController(private val dateListener: DateListener?) : CalendarController {
```

Comment:
Why listener can be null? Always prefer usage of non nullable types.

**This comment applies to all classes.**

Reply:
I don't want to force setting this listener, in some cases it could be unnecessary. Maybe more preferable solution will be empty constructor with setter?